package.json

```diff
@@ -3,13 +3,13 @@
   "description": "Material Components Web",
   "license": "Apache-2.0",
   "scripts": {
-    "build": "mkdirp build && webpack --progress --colors",
+    "build": "npm run clean && mkdirp build && webpack --progress --colors",
     "build:demo": "cross-env MDC_ENV=development MDC_BUILD_STATIC_DEMO_ASSETS=true MDC_WRAP_CSS_IN_JS=false npm run build",
     "build:min": "mkdirp build && cross-env MDC_ENV=production webpack -p --progress --colors",
     "changelog": "standard-changelog -i CHANGELOG.md -k packages/material-components-web/package.json -w",
-    "clean": "del-cli build/** build",
+    "clean": "del-cli build/** build .closure-tmp",
```

Comment:
You _might_ need to do:

`"clean": "del-cli build/** build .closure-tmp/** .closure-tmp"`

To force the directory to be fully deleted on macOS.